package.json

```diff
@@ -74,7 +74,6 @@
     "gh-pages": "^0.11.0",
     "gulp": "^3.9.1",
     "gulp-replace": "^0.5.4",
-    "gulp-util": "^3.0.8",
```

Comment:
Nice!  first dep gone!

Can you remove this with yarn so we clean up the lockfile as well?  Hopefully you can remove without updating the rest of the deps as that would be out of scope here.

Reply:
Yeah, I was running into issues with yarn. Which led me down this route: https://github.com/yarnpkg/yarn/issues/5962. I think our only solution is to remove the lockfile and reinstall (which unfortunately updates the rest of the deps). Maybe what I'll do is open a PR to be merged before this that fixes our lockfile.

Reply:
yarn.lock isn't being updated as this is still a transitive dependency from some other deps.  Still, it's days are numbered!